### PR TITLE
[fr] add antipattern to ACCORD_CENTAINE_DE

### PR DESCRIPTION
**Pull request by remymartinDev**

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -14933,6 +14933,11 @@ though, and has since been slightly modified)-->
       </rule>
     </rulegroup>
     <rule id="ACCORD_CENTAINE_DE" name="centaine de">
+      <antipattern>
+          <token inflected="yes">série</token>
+          <token>de</token>
+          <token>bande dessinée</token>
+      </antipattern>
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:14940]: We need 2 tokens: 
```
<token>bande</token>
<token>dessinée</token>
```
And a test sentence: `<example>série de bande dessinée</example>`

Comment (udomai) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:14940]: If you want to make a change to a pull request, for example in this case to split "bande dessinée" into two tokens, and to add an `<example>` to test if your AP works: just check out the branch on which you made the original PR (in this case, `RM-2`), make the changes, commit them and push to the same branch. You PR will be updated automatically.

**Comment by jaumeortola** [comment]
There are conflicts because the whole grammar.xml file was re-indented. The easiest thing to do is to rewrite the changes in the current master branch from scratch.

**Comment by udomai** [comment]
I'm going to merge that manually, it's only 6 lines 🙂

**Comment by udomai** [comment]
merged here: 08d53ce7901668ada3ea1b3d4a8a9b4842ffa550

**Pull request closed without merging**